Dockerfile

```diff
@@ -8,19 +8,20 @@ RUN yarn install --network-timeout 600000
 COPY src/ src
 COPY types/ types
 COPY nginx.conf .
-COPY public/ public/
 COPY tsconfig.json .
+COPY vite.config.ts .
 COPY . .
 RUN echo REACT_APP_GIT_SHA=`git rev-parse --short HEAD` >> .env.production
 RUN echo `git rev-parse --short HEAD` > health.html
+# TODO: Can we use yarn and add linting steps as well?
```

Comment:
Pls check this once

Reply:
It will increase the time of build. And anyways we are allowing the merge after lint so we can ignore here